torch/onnx/utils.py

```diff
@@ -468,7 +481,7 @@ def _model_to_graph(model, args, verbose=False,
         if not (isinstance(torch_out, list) or isinstance(torch_out, tuple)):
             output_wrapped = [torch_out]  # type: ignore
         else:
-            output_wrapped = torch_out  # type: ignore
+            output_wrapped = _remove_none_list(torch_out)  # type: ignore
 
         output_tensors, out_desc = torch._C._jit_flatten(tuple(output_wrapped))
         torch._C._jit_pass_onnx_assign_output_shape(graph, output_tensors, out_desc, _onnx_shape_inference)
```

Comment:
the `out_desc` afterwards is different, is that covered?

This doesn't feel compatible when support is added for None output from script module. Is there a design to be extensible for that?